plugins/destination/postgresql/client/migrate.go

```diff
@@ -63,8 +65,10 @@ func (c *Client) isTableExistSQL(ctx context.Context, table string) (bool, error
 
 func (c *Client) getStalePks(pgPKs map[string]bool, table *schema.Table) []string {
 	stalePks := []string{}
+	sortedPKs := maps.Keys(pgPKs)
+	sort.Strings(sortedPKs)
 	if c.enabledPks() {
```

Comment:
```suggestion
	if c.enabledPks() {
		sortedPKs := maps.Keys(pgPKs)
		sort.Strings(sortedPKs)
```

(nit, but I think this helps both efficiency and readability)

Reply:
Nice one 🚀